src/screens/benefit/Details.tsx

```
@@ -49,35 +48,45 @@
 }
 
 const BenefitsDetails: React.FC = () => {
   const { isOpen, onClose } = useDisclosure();
-  const [context, setContext] = useState<Context | null>(null);
+  const [context, setContext] = useState<FinancialSupportRequest | null>(null);
   const [item, setItem] = useState<BenefitItem | null>(null);
   const [loading, setLoading] = useState<boolean>(true);
   const [isApplied, setIsApplied] = useState<boolean>(false);
   const [error, setError] = useState<string>("");
-  const [authUser, setAuthUser] = useState<AuthUser | null>(null);
+  const [authUser, setAuthUser] = useState<Object | null>(null);
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)